Cargo.toml

```diff
@@ -311,7 +311,7 @@ spin = { version = "0.5.2", default-features = false }
 libc = { version = "0.2.48", default-features = false }
 
 [target.'cfg(any(target_os = "android", target_os = "freebsd", target_os = "linux", target_os = "netbsd", target_os = "openbsd", target_os = "solaris"))'.dependencies]
-lazy_static = { version = "1.3", default-features = false, optional = true }
+once_cell = { version = "1.2.0", default-features = false, features=["std"], optional = true }
```

Comment:
Is there any reason to do this when the only `default-features` of once_cell are `std`.

You could just do `once_cell = { version = "1.2.0", optional = true }`

Reply:
I always use `default-features = false` since doing this in almost every project I join cuts the dependencies by 10-25%. Although, the issues with `spin-rs` and similar when disabling crates' `std` default features may cause me to reconsider this policy.